source/configuring_domain/03_managing_tracking/third_party_analytics.rst

```diff
@@ -4,56 +4,45 @@ Third-party analytics
 The monitoring tools provided by Opendatasoft focus on API calls and usages.
 They are useful to gather precise information on how data is consumed (see :doc:`Analyzing domain usage </managing_domain/03_analyzing_domain_usage/index>`). But it can be limiting when trying to measure more broadly the audience of a portal: referral sources, visitors' geographical origin, time spent on the portal, etc.
 
-Third-party services, specialized in measuring audience, can give additional analytics such as those mentioned above. Opendatasoft supports integration with Google Analytics, and XiTi and SmartTag by AT Internet.
+Third-party services, specialized in measuring audience, can give additional analytics such as those mentioned above. Opendatasoft supports integration with Google Analytics, XiTi and SmartTag by AT Internet, and Matomo.
 
 All third-party analytics integrations are activated and configured from Configuration > Tracking in the back office.
 
 Once configured and running, they allow audience tracking on the front office of a domain.
 
 
-.. admonition:: Important
-   :class: important
-
-   For XiTi and SmartTag integration, Opendatasoft must be involved to activate the service before it can be configured in the back office.
-
-
 Google Analytics
 ----------------
 
-.. image:: images/configuration_tracking-google-analytics.png
-
 Google Analytics is an analytics solution edited by Google.
 
 Currently, only "Universal Analytics" properties are supported.
 
 To activate and configure Google Analytics:
 
 1. In the Configuration > Tracking section of the back office, go to the Google Analytics area.
```

Comment:
Not related to that particular PR, but in the legal section, "Configuration > Tracking" is displayed in bold but not in that section.

Reply:
[Fixed](https://github.com/opendatasoft/ods-documentation/pull/286/commits/f643f431a598413c8536af3dd5d083e689f7b7b6#diff-c6e440d6fd65214262f0083fffdade1a620fde20b4a04f2b43186c669a247b11)
Thanks for the review!